5/testing-genesis.md

```diff
@@ -0,0 +1,147 @@
+Testing Genesis
+===
+
+Right about meow, you may have noticed that the test setup can get quite tedious,
+i.e. having to create new kitties for each unit test.
+
+Wouldn't it be great to deploy our blockchain with some kitties already minted in the genesis block?
+
+Substrate lets you deploy your chain with preconfigured storage through a genesis configuration.
+
+In this section, we'll walk you through: 
+- Extending `decl_storage` to add extra genesis data
+- Mocking the genesis configuration in during testing
+- Testing that we have the correct genesis set up
+
+## Add Extra Genesis
+
+Let's start by importing some functions and types we'll be using to configure genesis.
+
+**substratekitties<span>.</span>rs**
+```rust
+use runtime_io::{with_storage, StorageOverlay, ChildrenStorageOverlay};
```

Comment:
Getting this error when compiling with Substrate v1.0:

```
error[E0432]: unresolved imports `runtime_io::with_storage`, `runtime_io::StorageOverlay`, `runtime_io::ChildrenStorageOverlay`
 --> /Users/shawntabrizi/Documents/GitHub/substrate-package/substrate-node-template/runtime/src/template.rs:8:18
  |
8 | use runtime_io::{with_storage, StorageOverlay, ChildrenStorageOverlay};
  |                  ^^^^^^^^^^^^  ^^^^^^^^^^^^^^  ^^^^^^^^^^^^^^^^^^^^^^ no `ChildrenStorageOverlay` in the root
  |                  |             |
  |                  |             no `StorageOverlay` in the root
  |                  no `with_storage` in the root
  |                  help: a similar name exists in the module: `set_storage`
```

Reply:
I have a working version in this branch: https://github.com/paritytech/substrate/tree/kiz-document-kitty. I will try to recreate it with substrate-package and see if I get the same issue.

Reply:
So this probably uses master branch right?

Switch to v1.0 and you will likely run into the same issue. Possibly we need to backport whatever changes cause issues to the v1.0 branch, maybe open an issue?